evalml/pipelines/components/transformers/preprocessing/time_series_featurizer.py

```diff
@@ -246,7 +249,7 @@ def _compute_delays(self, X_ww, y):
                     feature_name = f"{col_name}_delay_{self.start_delay + t}"
                     lagged_features[
                         f"{col_name}_delay_{self.start_delay + t}"
-                    ] = col.shift(self.start_delay + t)
+                    ] = col.shift(t)
```

Comment:
lags already have the start delay baked in since we're only selecting lags that start from the feature engineering window.